sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/CustomEvents/CustomEventLogger.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Monitor.OpenTelemetry.CustomEvents
+{
+    /// <summary>
+    /// Default logger for logging custom events telemetry.
+    /// </summary>
+    public sealed class CustomEventLogger : ICustomEventLogger
+    {
+        private static readonly Func<IReadOnlyList<KeyValuePair<string, object?>>?, Exception?, string> s_formatter = (state, ex) =>
+        {
+            return "CustomEvent";
+        };
+
+        private readonly ILogger _logger;
+
+        /// <summary>
+        /// Custom events.
+        /// </summary>
+        /// <param name="loggerFactory">LoggerFactory instance for logging events.</param>
+        /// <exception cref="ArgumentNullException"></exception>
+        public CustomEventLogger(ILoggerFactory loggerFactory)
+        {
+            if (loggerFactory == null)
+                throw new ArgumentNullException(nameof(loggerFactory));
+
+            _logger = loggerFactory.CreateLogger("Azure.Monitor.OpenTelemetry.CustomEvents");
+        }
+
+        /// <inheritdoc/>
+        public void TrackEvent(string name, IReadOnlyList<KeyValuePair<string, object?>>? attributes = null)
+        {
+            // Setting the loglevel to critical to ensure events are not filtered.
```

Comment:
Additionally specifying the loglevel is not honored for custom events in the comments would help.